CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 You can find all notable changes to Explorer for Endevor in this document.
 
+## [1.0.0] &ndash; 2022-02-23
+
+### Added
+
+- Telemetry event recording.
+- Added a `[MAP]` folder in the tree view. It is being populated with elements found up the map.
+- Added a button `Show logs` in the notification panel which allows the user to open the extension's output.
+
+### Changed
+
+- Multiple error logging and notification events.
+
+### Fixed
+
+- Fixed an issue in the URI building process. Characters that may potentially break the building process are now being escaped.
+- Upload dialogs now prevent wildcards as input.
+- Fixed an issue with conflict resolution causing the command to fail when resolving reoccurring conflicts.
+- Fixed an issue with retrieving, editing multiple elements with auto signout option. Now it processes all the selected elements in groups by each location profile.
```

Comment:
```suggestion
- Fixed an issue with retrieving and editing multiple elements with the auto signout option. Now auto signout processes all the selected elements in groups by location profile.
```

Reply:
@roman-kupriyanov That sounds good to me. Great suggestion

Reply:
```suggestion
- Fixed an issue with retrieving and editing multiple elements with the auto signout option. Now the retrieve and edit commands correctly process all the selected elements in groups by location profiles.
```